src/js/disability-benefits/reducers/claims-list.js

```diff
@@ -22,19 +22,30 @@ function phaseChangeDate(claim) {
 export default function claimsReducer(state = initialState, action) {
   switch (action.type) {
     case SET_CLAIMS: {
-      const sortedList = _.orderBy([phaseChangeDate, 'id'], 'desc', action.claims);
+      return _.assign(state, {
+        list: action.claims
+      });
+    }
+    case FILTER_CLAIMS: {
+      let sortedList = _.orderBy([phaseChangeDate, 'id'], 'desc', state.list);
```

Comment:
In the next feature, custom sort options will be added, so it made sense to separate sorting from setting the claims list, I kept it with the filtering for now but may separate it in next PR